docs/applications/nube-sdk/components/box.md

```
@@ -8,10 +8,10 @@ import AppTypes from '@site/src/components/AppTypes';
 Box container, supports multiple sizes, styling and alignment options.
 It's used mostly to embed other components that don't have their own layout options (fields, texts, images, etc..)
 
-### Usage
-
 ![Box](../../../../static/img/pt/nube-sdk-ui-box-1.png "Box")
 
+### Usage
```

markdownlint-cli2 (0.20.0)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)